pipeline.yml

```diff
@@ -868,13 +859,13 @@ resources:
     secrets_passphrase: {{development-secrets-passphrase}}
     bosh_cert: bosh-tooling.pem
 
-- name: secrets-common
-  type: cg-common
-  source:
-    bucket_name: {{common-bucket-name}}
-    region: {{aws-region}}
-    secrets_file: {{common-secrets-file}}
-    secrets_passphrase: {{common-secrets-secrets-passphrase}}
+#- name: secrets-master
+#  type: cg-common
+#  source:
+#    bucket_name: {{master-bucket-name}}
+#    region: {{aws-region}}
+#    secrets_file: {{master-secrets-file}}
+#    secrets_passphrase: {{master-secrets-secrets-passphrase}}
```

Comment:
Any reason this is commented with the name change?

Reply:
masterbosh isn't using updated secrets process yet and the earlier `secrets-common` resource is no longer used across all deployments

Reply:
Does that matter? A bucket, region, filename, and passphrase should all exist, yeah? Isn't this: https://github.com/18F/cg-secret-rotation/pull/6/files#diff-4e39b6ee47adc972ea04e30a02d2b1e2R152 And if not, why not? Also, this resource isn't referenced anywhere else in the pipeline as of yet.

Reply:
It is the same bucket, region, filename, and passphrase used in https://github.com/18F/cg-secret-rotation/blob/b7aaea2289a25ba5ae37694b7dbc51788a3410d8/ci/pipeline.yml#L217, only commented out _because_ `deploy-master-bosh` doesn't use secrets from there yet.